django/db/models/fields/files.py

```diff
@@ -75,6 +75,7 @@ def open(self, mode='rb'):
             self.file.open(mode)
         else:
             self.file = self.storage.open(self.name, mode)
+        return self
```

Comment:
Yes, please create another pull request. You can use "Refs #27777 -- ..." for the commit message.